### DISCUSSION TITLE
Issues with `async_trait`

### DISCUSSION THREAD

**Discussion by InAnYan** [Abraxas-365/langchain-rust#263] [General]
I once written a Rust library that used traits and `async_trait` extensively, and quickly ran into problems from this thread:

https://github.com/rust-lang/rust/issues/110338

How is the development of `langchain-rust` going? No issues like that?

Does anyone already used `langchain-rust`, did you had issues like that?

**Comment by prabirshrestha** [comment]
There are no issues.
